include/http.h

```diff
@@ -80,6 +82,12 @@ typedef struct http_link_context_s
     TYPE_REQUEST,
     TYPE_WEBSOCKET
   } type;
+  bool first_response;
+
+  // Data for logging
+  char peer_ip[45];
```

Comment:
17 is enough for ip.

Reply:
45 is needded for IPv6. I know we currently use only IPv4 and not everything is ready for IPv6, but if we want to make it work in the future we wouldn't need to change this.